justfile

```
@@ -25,3 +25,7 @@ _rsync_ubuntu:
 
 _port_forward_ubuntu:
     ssh -L 8181:localhost:8181 ubuntu -N
+
+[no-cd]
+expand *args:
```

nit: what about

```suggestion
_expand *args:
```

to make it not publicly visible, since this may not be a standard procedure but seems like ad-hoc helpful tools